guideGlobalMetadata.json

```
@@ -3,6 +3,8 @@
         "SpikeGLXRecordingInterface",
         "PhySortingInterface",
         "NeuroScopeSortingInterface",
-        "BiocamRecordingInterface"
+        "BiocamRecordingInterface",
+        "Spike2RecordingInterface",
+        "CEDRecordingInterface"
```

```suggestion
        "Spike2RecordingInterface",
```

CEDRecordingInterface = Spike2RecordingInteface

However, CEDRecordingInterface is the deprecated version of the name: https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/datainterfaces/ecephys/spike2/spike2datainterface.py#L76-L79